CODE_OF_CONDUCT.md

```diff
@@ -40,7 +39,7 @@ Project maintainers are responsible for clarifying the standards of acceptable
 
 Project maintainers have the right and responsibility to remove, edit, or
  reject comments, commits, code, wiki edits, issues, and other contributions
- that are not aligned to this Code of Conduct, or to ban
+ that are not aligned to this Code of Conduct, or ban
```

Comment:
The "to" here is helpful. It is the right and responsibility of maintainers... "to remove...., or to ban....", in my opinion. We can rephrase though if it's confusing.

Reply:
I think "or ban temporarily or permanently any contributor", sounds better, although using "to" doesn't really make any changes to the meaning, but it is just that, why use an extra preposition when we can make it mean the same with fewer words.

Reply:
The ' or to ' is in context of 'having the right and responsibility...' - 'having the right and responsibility ban' is not a correct sentence, 'having the right and responsibility to ban' is. The 'to' should stay.